### ISSUE TITLE
There are no commands defined in the "backpack:base" namespace.

### ISSUE THREAD

**Issue by nileshd** [Laravel-Backpack/CRUD#1615]
# Bug report
When i try to run php artisan backpack:base:install, it says "There are no commands defined in the "backpack:base" namespace." and would not proceed.

### What I did:
I installed backpack/crud 
```
root@12bfba793ca0:/APP/src# composer require backpack/crud
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Using version ^3.4 for backpack/crud
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 7 installs, 0 updates, 0 removals
  - Installing composer/installers (v1.5.0): Downloading (100%)         
  - Installing venturecraft/revisionable (1.28.0): Downloading (100%)         
  - Installing jenssegers/date (v3.4.0): Downloading (100%)         
  - Installing prologue/alerts (0.4.2): Downloading (100%)         
  - Installing almasaeed2010/adminlte (v2.4.3): Downloading (100%)         
  - Installing backpack/base (0.9.7): Downloading (100%)         
  - Installing backpack/crud (3.4.30): Downloading (100%)         
Writing lock file
Generating optimized autoload files

```
and then I tried to run php artisan backpack:base:install
```
root@12bfba793ca0:/APP/src# php artisan backpack:base:install

     There are no commands defined in the "backpack:base" namespace.  
 
```                                                                  

### What I expected to happen:
The command would install the backpack:base

### What happened:
It could not find the command. Backpack is in the vendor directory, but it did not register itself in my app as such nor put a config/backpack/base.php in the config directory.

### Backpack, Laravel, PHP, DB version:
Backpack: 3.4.30
Laravel: 5.6.29
PHP: 7.2.5

**Comment by nileshd** [comment]
I also tried the manual installation/publishing

`root@12bfba793ca0:/APP/src# php artisan vendor:publish --provider="Backpack\Base\BaseServiceProvider" --tag="minimum"
Publishing complete.
`
But nothing actually gets published, even though it says publishing complete.

So, I'd think that's probably where the problem might be. It's installing, but not able to publish to my own app somehow.

**Comment by tabacitu** [comment]
Hmmm what Laravel version did  you install it on? After running ```composer require backpack/crud``` did Laravel do its auto-registering magic? Did it tell you in the command line that it registered a bunch of packages (Backpack included)? That might be the issue.

**Comment by nileshd** [comment]
I installed it on Laravel version 5.6.29 but it did not do the auto-registering magic. Composer just put the packages in the vendor directory and that was it.

I just manually added the base and crud packages to the config/app.php and copies the config/backpack folders to config and now it's working fine. 

Not really sure if it's a problem with laravel 5.6.29 to not autoregister or something with my install.
